CONTRIBUTING.md

```
@@ -177,6 +184,38 @@ templates: Update board layout for mobile responsiveness.
+### Django Tests
+```bash
+python manage.py test game --verbosity=2
+```
+
+### Selenium E2E Tests
+```bash
+python manage.py test game.selenium_tests --verbosity=2
+```
+
+### Checking for Missing Migrations
+```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)